university-of-new-england-australia-note.csl

```diff
@@ -0,0 +1,558 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>University of New England, Australia (note)</title>
+    <id>http://www.zotero.org/styles/university-of-new-england-australia-note</id>
+    <link href="http://www.zotero.org/styles/university-of-new-england-australia-note" rel="self"/>
+    <link href="http://www.zotero.org/styles/universita-di-bologna-lettere" rel="template"/>
+    <link href="http://www.une.edu.au/__data/assets/pdf_file/0007/87811/REF_Trad.-Footnoting.pdf" rel="documentation"/>
+    <author>
+      <name>Benjamin Hollister</name>
+      <email>ben@rootsandbranches.com.au</email>
+      <uri>http://www.mendeley.com/profiles/benjamin-hollister2/</uri>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <category field="history"/>
+    <summary>University of New England (Australia) Discipline of History Traditional Footnoting</summary>
+    <updated>2015-09-05T21:56:36+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor-translator">
+    <group delimiter=", ">
+      <choose>
+        <if variable="author">
+          <names variable="editor" delimiter=", ">
+            <name delimiter=", "/>
+            <label form="verb" prefix=" (" suffix=") "/>
```

Comment:
is this the one you didn't change? Because "Firstname Editor (edited by)" still seems very weird to me. You can just change form to long to get editor/editors